src/librustdoc/theme.rs

```diff
@@ -103,16 +103,16 @@ fn is_line_comment(pos: usize, v: &[u8], events: &[Events]) -> bool {
     if let Some(&Events::StartComment(_)) = events.last() {
         return false;
     }
-    pos + 1 < v.len() && v[pos + 1] == b'/'
+    v[pos + 1] == b'/'
```

Comment:
This is going to panic if `is_line_comment()` is called with `pos +1 >= v.len()` - is the introduction of that panic intentional or is this fallout from fixing the tests in `load_css_events()` ?

Reply:
Would `v.get(pos+1) == Some(b'/')` be acceptable if you want to avoid the `pos + 1 < v.len()`?

Reply:
Normally it shouldn't be possible to be called with such an issue arising thanks to the change line 113:

```rust
while pos + 1 < v.len() {
```

However, this is a good point and I think I'll update the code to what you suggested.

Reply:
Based on high-five, looks like my suggestion missed a borrow :-/